examples/bri-3/.env.sample

```diff
@@ -19,3 +19,4 @@ BPI_ENCRYPTION_KEY_KTY_PARAM="oct"
 SNARKJS_CIRCUITS_PATH="zeroKnowledgeArtifacts/circuits/"
 VSM_CYCLE_PERIOD_IN_SECS=15
 VSM_CYCLE_TX_BATCH_SIZE=5
+HASH_ALGORITHM="sha256"
```

Comment:
i think it would be better to have a more specific name here. For example, MERKLE_TREE_HASH_ALGH or something like that. Makes it easier to put the var in context.